webdriver/tests/bidi/script/get_realms/context.py

```diff
@@ -29,17 +27,14 @@ async def test_context(
 
     result = await bidi_session.script.get_realms(context=new_context["context"])
 
-    recursive_compare(
-        [
-            {
-                "context": new_context["context"],
-                "origin": test_origin,
-                "realm": new_context_result["realm"],
-                "type": "window",
-            },
-        ],
-        result,
-    )
+    assert [
+        {
+            "context": new_context["context"],
+            "origin": test_origin,
+            "realm": new_context_result["realm"],
+            "type": "window",
+        },
+    ] <= result
```

Comment:
Hm, interesting. Now this assertion allows having more then 1 realm in the response, which can actually be correct.